### ISSUE TITLE
Create self-hosting guide

### ISSUE THREAD

**Issue by matt-aitken** [triggerdotdev/trigger.dev#48]
The guide should make it easy to host the project yourself.

Trigger.dev involves several services. 

Our current deployment system:
- The main web app and web socket services using Flight Control (see [flightcontrol.json].(https://github.com/triggerdotdev/trigger.dev/blob/bf57430b6663fa62fc4a395d3aeb05104eae0d1b/flightcontrol.json)
- The authentication service (which uses Nango) to fly.io.
- Apache Pulsar using [StreamNative](https://streamnative.io/product).

**Comment by matt-aitken** [comment]
Please note that there is a [development guide](https://github.com/triggerdotdev/trigger.dev/blob/bf57430b6663fa62fc4a395d3aeb05104eae0d1b/DEVELOPMENT.md) if you want to run locally.

**Comment by kryptonianson** [comment]
Looking forward to this. Docker containers would be fantastic.

**Comment by cmolina** [comment]
Consider using Nix 🙏🏻

**Comment by jeanpfs** [comment]
Is this not going to happen anymore?

**Comment by ericallam** [comment]
@jeanpfs yes, this is definitely going to happen! We're just coordinating these guides (we plan on having a guide for as many platforms as possible) on our [trigger.dev-docs](https://github.com/triggerdotdev/trigger.dev-docs) repo
